src/libs/ajax.js

```diff
@@ -719,6 +719,22 @@ export const Summary = {
   }
 };
 
+export const Support = {
+
+  //fetchany
+  //Noty
+  createSupportRequest: async (ticket) => {
+    const res = await fetchOk(`https://broadinstitute.zendesk.com/api/v2/requests.json`, fp.mergeAll([Config.jsonBody(ticket), { method: 'POST' }]));
+    return await res.json();
+  },
+
+  //Create File Name
+  uploadAttachment: async (file) => {
+    const res = await fetchOk(`https://broadinstitute.zendesk.com/api/v2/uploads?filename=${file.name}`, fp.mergeAll([Config.attachmentBody(file), { method: 'POST' }]));
```

Comment:
Have not tested this yet, but there is a potential issue if `file.name` is not url encoded safely or has strange characters. I can imagine some weird things happening with unicode characters. As discussed in review session, would probably be safer to generate a file name instead of using what was passed in.

Reply:
I made the name default to "Attachment" since the token will always be unique